pages/seller/js/seller.js

```diff
@@ -29,6 +45,9 @@ let products = [
 ];
 
 
+//get index--------------------
+let index = products.length;
```

Comment:
Not `index` should be named like `productSize` or `productLength`

Reply:
Yes teacher